client/app/bundles/course/assessment/pages/AssessmentStatistics/LiveFeedbackHistory/LiveFeedbackHistoryPage.tsx

```diff
@@ -19,6 +19,10 @@ const translations = defineMessages({
     id: 'course.assessment.liveFeedback.questionTitle',
     defaultMessage: 'Question {index}',
   },
+  feedbackTimingTitle: {
```

Comment:
please do this change inside https://github.com/Coursemology/coursemology2/pull/7497/commits/6837a5b595e7d2a58f3c4c3a63cb35ed2d24a84e instead of creating the new one

Reply:
done